### ISSUE TITLE
Open-match fails to setup on minikube

### ISSUE THREAD

**Issue by abarnes07** [googleforgames/open-match#978]
Hello,

The documentation provided on https://open-match.dev/site/docs/installation/install_om/ seems to indicate open-match works on minikube. When following through with the installation, there seems to be an issue in the v0.7.0/yaml/install.yaml file out of the box in regards to setting up grafana, which causes the installation to fail:
Output of: `kubectl apply -f https://open-match.dev/install/v0.7.0/yaml/install.yaml --namespace open-match`

```
Error from server (Invalid): error when applying patch:
{"spec":{"storageClassName":null}}
to:
Resource: "/v1, Resource=persistentvolumeclaims", GroupVersionKind: "/v1, Kind=PersistentVolumeClaim"
Name: "open-match-grafana", Namespace: "open-match"
Object: &{map["apiVersion":"v1" "kind":"PersistentVolumeClaim" "metadata":map["annotations":map["control-plane.alpha.kubernetes.io/leader":"{\"holderIdentity\":\"b6d775c4-06fe-11ea-be66-08002794bdbe\",\"leaseDurationSeconds\":15,\"acquireTime\":\"2019-11-14T20:09:37Z\",\"renewTime\":\"2019-11-14T20:10:08Z\",\"leaderTransitions\":0}" "kubectl.kubernetes.io/last-applied-configuration":"{\"apiVersion\":\"v1\",\"kind\":\"PersistentVolumeClaim\",\"metadata\":{\"annotations\":{},\"labels\":{\"app\":\"grafana\",\"chart\":\"grafana-3.5.7\",\"heritage\":\"Tiller\",\"release\":\"open-match\"},\"name\":\"open-match-grafana\",\"namespace\":\"open-match\"},\"spec\":{\"accessModes\":[\"ReadWriteOnce\"],\"resources\":{\"requests\":{\"storage\":\"10Gi\"}},\"storageClassName\":null}}\n" "pv.kubernetes.io/bind-completed":"yes" "pv.kubernetes.io/bound-by-controller":"yes" "volume.beta.kubernetes.io/storage-provisioner":"k8s.io/minikube-hostpath"] "creationTimestamp":"2019-11-14T20:09:37Z" "finalizers":["kubernetes.io/pvc-protection"] "labels":map["app":"grafana" "chart":"grafana-3.5.7" "heritage":"Tiller" "release":"open-match"] "name":"open-match-grafana" "namespace":"open-match" "resourceVersion":"18012" "selfLink":"/api/v1/namespaces/open-match/persistentvolumeclaims/open-match-grafana" "uid":"817016f2-b1cc-486d-80e4-de2bb4add753"] "spec":map["accessModes":["ReadWriteOnce"] "resources":map["requests":map["storage":"10Gi"]] "storageClassName":"standard" "volumeMode":"Filesystem" "volumeName":"pvc-817016f2-b1cc-486d-80e4-de2bb4add753"] "status":map["accessModes":["ReadWriteOnce"] "capacity":map["storage":"10Gi"] "phase":"Bound"]]}
for: "https://open-match.dev/install/v0.7.0/yaml/install.yaml": PersistentVolumeClaim "open-match-grafana" is invalid: spec: Forbidden: is immutable after creation except resources.requests for bound claims
[unable to recognize "https://open-match.dev/install/v0.7.0/yaml/install.yaml": no matches for kind "PodSecurityPolicy" in version "extensions/v1beta1", unable to recognize "https://open-match.dev/install/v0.7.0/yaml/install.yaml": no matches for kind "Deployment" in version "extensions/v1beta1", unable to recognize "https://open-match.dev/install/v0.7.0/yaml/install.yaml": no matches for kind "DaemonSet" in version "extensions/v1beta1", unable to recognize "https://open-match.dev/install/v0.7.0/yaml/install.yaml": no matches for kind "Deployment" in version "apps/v1beta2", unable to recognize "https://open-match.dev/install/v0.7.0/yaml/install.yaml": no matches for kind "StatefulSet" in version "apps/v1beta2"]
```

Kubernetes Version:
```
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.3", GitCommit:"b3cbbae08ec52a7fc73d334838e18d17e8512749", GitTreeState:"clean", BuildDate:"2019-11-13T11:23:11Z", GoVersion:"go1.12.12", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.2", GitCommit:"c97fe5036ef3df2967d086711e6c0c405941e14b", GitTreeState:"clean", BuildDate:"2019-10-15T19:09:08Z", GoVersion:"go1.12.10", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by yfei1** [comment]
Hi @abarnes07,

Thanks for using Open Match and I am really sorry about the confusion. Our team noticed this bug and fixed it in our master branch. This fix is included in the v0.8.0 release that we just published few minutes ago and Open Match should work as expected under Minikube environment.

Just a friendly reminder that we have updated our website open-match.dev with a bunch of new resources to help you on-board. Feedback are welcome and I hope the update could resolve your issue :)
